libkbfs/interfaces.go

```diff
@@ -1679,6 +1695,9 @@ type MutableBareRootMetadata interface {
 	// key generation.
 	GetUserDeviceKeyInfoMaps(keyGen KeyGen, extra ExtraMetadata) (
 		readers, writers UserDeviceKeyInfoMap, err error)
+	// OnRekeyDone is called after all rekeying work has been performed on the underlying
```

Comment:
Re: `Crypto` vs. `Config`. I actually have that change coming in the next PR for rekeys. Mind if I wait for that?